Dockerfile

```diff
@@ -11,21 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:1.12.9 AS build
 
+FROM alpine:3.9
 MAINTAINER Timothy St. Clair "tstclair@heptio.com"  
-WORKDIR /src/
-COPY . .
 
-RUN CGO_ENABLED=0 go build -o /src/eventrouter
-
-FROM alpine:3.9 as release
-
-WORKDIR /app
 RUN apk update --no-cache && apk add ca-certificates
-COPY --from=build /src/eventrouter .
+ADD eventrouter /eventrouter 
```

Comment:
So something is wrong with the image that gets built.

I'm able to build it OK but then it fails to run:
```
$ kubectl logs eventrouter-854dbfdc6d-dt7lv -n kube-system
/bin/sh: /eventrouter: not found
```

When I run the image and look inside it is there:
```
$ pwd
/
$ ls -lah|grep event
-rwxr-xr-x    1 root     root       53.6M Sep 27 20:10 eventrouter
```

I think one issue is that you're putting it into the root directory but we are running as `nobody:nobody`, but even after I manually run the image as a root user I still get the same errors.

I tried moving it into `/app/eventrouter` as well and the same issues. I can see it in `ls`, I can even `cat ./eventrouter|head` and `cp ./eventrouter foo` but I can not run the files. I can `chown` them and `chmod +x ...` them as the root user, but still not run them.

Reply:
I found the issue; see comment in makefile.

Reply:
fixed the Makefile. I have also moved `/eventrouter` to `/app/eventrouter`